app/views/devise/shared/_links.html.haml

```diff
@@ -1,5 +1,5 @@
 .auth-support-links
-  - if controller_name != 'sessions'
+  - if not ['sessions', 'root'].include?(controller_name)
```

Comment:
to make this more readable, let's use `- unless controller_name.in?(['sessions', 'root'])`